litgpt/finetune/lora.py

```diff
@@ -94,8 +94,8 @@ def setup(
     pprint(locals())
     data = Alpaca() if data is None else data
     devices = parse_devices(devices)
-    config = Config.from_name(
-        name=checkpoint_dir.name,
+    config = Config.from_file(
+        checkpoint_dir / "model_config.yaml",
```

Comment:
When the user passe `model_name`, `checkpoint_dir.name` will not exist in the config dictionaries, so this is why it's important to read the config from a file instead

Reply:
There was a use case here why we did checkpoint_dir.name, but I don't remember. Maybe it was because if you did pretraining and then finetune a checkpoint, it wouldn't have a model config file in the directory.